internal/cluster/validations/validations.go

```diff
@@ -268,10 +268,14 @@ func ValidateHTTPProxyFormat(proxyURL string) error {
 // ValidateNoProxyFormat validates the no-proxy format which should be a comma-separated list
 // of destination domain names, domains, IP addresses or other network CIDRs. A domain can be
 // prefaced with '.' to include all subdomains of that domain.
-// Use '*' to bypass proxy for all destinations.
-func ValidateNoProxyFormat(noProxy string) error {
+// Use '*' to bypass proxy for all destinations in OCP 4.8 or later.
+func ValidateNoProxyFormat(noProxy string, ocpVersion string) error {
 	if noProxy == "*" {
-		// TODO: Start accepting '*' when https://bugzilla.redhat.com/show_bug.cgi?id=1947066 is fixed, return an error until then
+		if wildcardSupported, err := common.VersionGreaterOrEqual(ocpVersion, "4.8.0-fc.4"); err != nil {
```

Comment:
Wouldn't "4.8" be sufficient ? 
How would you suggest to track the need to remove this code once we'll support OCP 4.8 and above ?

Reply:
It will fail, because 4.8 is a release version while 4.8.0-fc.4 is a pre-release.

Reply:
Try changing to `common.VersionGreaterOrEqual(ocpVersion, "4.8")` and see what happens to the unit-tests ;)

Reply:
@YuviGold do you see any reason to be concerned by comparing to the pre-release version in the meantime ?

Reply:
I guess that's fine for now. Once the release will be available, it should be changed since pre-releases won't be supported.

Reply:
It won't matter. This condition covers everything after `4.8.0-fc.4`, including later pre-release and releases starting with `4.8.0`. Look at this test case https://github.com/openshift/assisted-service/pull/1957/files#diff-142e4e6d84ff9769edd6c1e72b908d885324d62da8048f25f24dcb8f86fbbb42R316

Reply:
/lgtm